### DISCUSSION TITLE
Easy bromide webpage fix

### DISCUSSION THREAD

**Discussion by chararchter** [bromite/bromite#2553] [Ideas]
Here is how to make the bromide webpage https://www.bromite.org/fdroid more user-friendly:

Fdroid asks for address and fingerprint while installing, and right now the website offers both in one code block. The problem is **it's hard to select and copy only part of the text**, especially while using a phone. Splitting those pieces of information solves this.

## Current website:
<p>Always verify that the repository details are exactly:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Address: https://fdroid.bromite.org/fdroid/repo
Fingerprint of the signing key: E1EE5CD076D7B0DC84CB2B45FB78B86DF2EB39A3B6C56BA3DC292A5E0C3B9504
</code></pre></div></div>

```html
<p>Always verify that the repository details are exactly:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Address: https://fdroid.bromite.org/fdroid/repo
Fingerprint of the signing key: E1EE5CD076D7B0DC84CB2B45FB78B86DF2EB39A3B6C56BA3DC292A5E0C3B9504
</code></pre></div></div>
```

## Proposed update:
<p>Always verify that the repository details are exactly:</p>
<p>Address:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>https://fdroid.bromite.org/fdroid/repo
</code></pre></div></div>
<p>Fingerprint of the signing key:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>E1EE5CD076D7B0DC84CB2B45FB78B86DF2EB39A3B6C56BA3DC292A5E0C3B9504
</code></pre></div></div>

```html
<p>Always verify that the repository details are exactly:</p>
<p>Address:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>https://fdroid.bromite.org/fdroid/repo
</code></pre></div></div>
<p>Fingerprint of the signing key:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>E1EE5CD076D7B0DC84CB2B45FB78B86DF2EB39A3B6C56BA3DC292A5E0C3B9504
</code></pre></div></div>
```

## Copy to clipboard button
Bonus points for adding **copy to clipboard** button, as fingerprint of the key is wider than a phone screen, could look like this:

![copy-to-clipboard-button](https://user-images.githubusercontent.com/30589284/212930964-4314aee1-f412-4422-85c1-3ba4f9d8f5f0.png)

**Comment by danieljarolim** [comment]
Copy the link from the QR code. It gives you the fingerprint as part of the URL and f-droid will pick up both. It should be made clearer on page.

**Comment by Universalizer** [comment]
Webpage fix, display long text wrapping isn't it.❓

Simultaneously, link to either copy and paste or directly open ?

### Webpage fixing :-

<p>Simplifying text recognition :-
<a href="https://fdroid.bromite.org/fdroid/repo?fingerprint=E1EE5CD076D7B0DC84CB2B45FB78B86DF2EB39A3B6C56BA3DC292A5E0C3B9504">This is Bromite's full F-Droid repository address with Fingerprint of the signing key :-<br>
<br>https://fdroid.bromite.org/fdroid/repo?fingerprint=<br>E1EE5CD076D7B0DC84CB2B45FB78B86D<br>F2EB39A3B6C56BA3DC292A5E0C3B9504</a></p>

<hr><hr>
Possible try this "same above link" following :-

<a href="https://fdroid.bromite.org/fdroid/repo?fingerprint=E1EE5CD076D7B0DC84CB2B45FB78B86DF2EB39A3B6C56BA3DC292A5E0C3B9504">This is Bromite's F-Droid Repository with Fingerprint of the signing key, </a>

<p>Either ; 1) Directly Click and link to < System Prompt "open with" > Select and confirm your F-Droid app,<br>
<br>OR 2) Simply "Copy link address" "and "Paste in F-Droid."</p>

@danieljarolim, is also right.
